<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchWindow</class>
 <widget class="QWidget" name="BatchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="watchlistItemsTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>421</width>
     <height>371</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Item Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total Cost</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string># Crafts</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="watchlistItemsLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Watchlist Items</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="shoppingListTable">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>Item Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="shoppingListLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Shopping List</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="estTotalInvestmentLineEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>380</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="estTotalInvestmentLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Estimated Total Investment</string>
   </property>
  </widget>
  <widget class="QLabel" name="estTotalProfitLabel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>360</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Estimated Total Profit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="estTotalProfitLineEdit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>380</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="totalCraftsLabel">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>360</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Crafts</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalCraftsLineEdit">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>380</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>380</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="preventOverflowCheckBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Adjusts item quantities so there is no leftover materials.</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="text">
    <string>Prevent Overflow</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="overflowTable">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>40</y>
     <width>221</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>Item Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="overflowLabel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Overflow</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
